questionnaires/superset/README.md

```diff
@@ -112,3 +112,46 @@ docker-compose up
 ```
 docker-compose up
 ```
+
+## Share Dashboard
+
+### How to Share Dashboard Publicly
```

Comment:
### How to Share a Dataset Publicly

This is a pre-requisite to publicly sharing your dashboard. Keep in mind that when you make this dataset public, hypothetically any of the data can be exposed through a dashboard. After you share your dashboard, which uses part of the data, someone may come along and create a new dashboard from your dataset - they may or may not notice that your dataset is public - which publicly exposes other parts of your dataset that you didn't expect.

For this reason, it's strongly recommended to

1. Make a copy of your original private dataset, and label it "PUBLIC_Dataset A"
2. Limit the data in this dataset to only what you need for your dashboard visualizations
3. Adjust your dashboard to use this public dataset instead of your original private dataset